packages/block-editor/src/private-apis.js

```diff
@@ -93,4 +94,5 @@ lock( privateApis, {
 	useBlockDisplayTitle,
 	__unstableBlockStyleVariationOverridesWithConfig,
 	setBackgroundStyleDefaults,
+	useBlockBindingsUtils,
```

Comment:
What's the rationale for exposing `useBlockBindingsUtils` as a private API given that it isn't used outside of the package where it is defined?